app/javascript/components/mentoring/sorter.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
This component is not specific to mentoring I think and I assume there will be more places where it will be used. When that happens, we should probably move this to the parent `components` directory. The same goes for the pagination and textfilter components. For now, this is perfectly fine.

Reply:
Agreed. Now that I'm working on the `<TrackList />`, I will extract this to a more generic component.